cd_scripts/run.sh

```diff
@@ -37,20 +37,37 @@ export CF_SPACE=staging
 export APP_HOST=${APP_HOST_STAGING}
 deploy_application
 
+echo "Creating temporary route for integration tests"
+create_random_route_name
+HTBHF_APP="help-to-buy-healthy-foods-${CF_SPACE}"
+cf map-route ${HTBHF_APP} ${CF_PUBLIC_DOMAIN} --hostname ${ROUTE}
 
-if [ "$RUN_COMPATIBILITY_TESTS" == "true" ]; then
-    export RUN_TESTS="true"
+prepare_web_tests
+
+echo "Running integration tests"
```

Comment:
This should be inside an if statement. Otherwise it'll get run when we make changes to the cd script.